### ISSUE TITLE
-std=c++11 flag not working

### ISSUE THREAD

**Issue by alfranc** [hebaishi/linter-gcc#74]
Hi,
I am new to c++ and learning the language from the book "Programming: Principles and Practice Using C++ " by Bjarne Stroustrup. He uses a header file "std_lib_facilities.h". When I open this header file in atom, a 20 something errors appear.But when I compile it using command prompt with c++11 flag no errors are shown.Also when i include this file it says the file does not exist,although its present with the source file in the folder
Thanks in advance.

![std](https://cloud.githubusercontent.com/assets/21018348/17647882/2435bb0c-6221-11e6-81f1-325d8076f241.png)

![stdhead](https://cloud.githubusercontent.com/assets/21018348/17647885/31e20f1c-6221-11e6-962c-8a61208474bc.png)

![folder](https://cloud.githubusercontent.com/assets/21018348/17647891/3e0b3a02-6221-11e6-98de-ee03bdc4a7d0.png)

**Comment by hebaishi** [comment]
Looks like you just need to add ```.``` to your ```gccIncludePaths``` so your header file is picked up.

**Comment by alfranc** [comment]
Thanks a lot,Hebaishi. The issue is fixed.But what about errors coming in "std_lib_facilities.h".Can you please look into that too.

**Comment by hebaishi** [comment]
That does not look like a standard header file to me. This is a problem with your code and not with linter-gcc itself.

**Comment by alfranc** [comment]
Thanks a lot, Hebaishi.
